packages/kbn-securitysolution-io-ts-utils/BUILD.bazel

```diff
@@ -0,0 +1,89 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config", "ts_project")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library", "pkg_npm")
+
+PKG_BASE_NAME = "kbn-securitysolution-io-ts-utils"
+PKG_REQUIRE_NAME = "@kbn/securitysolution-io-ts-utils"
+
+SOURCE_FILES = glob(
+  [
+    "src/**/*.ts",
+  ],
+  exclude = [
+    "**/*.test.*"
+  ],
+)
+
+SRCS = SOURCE_FILES
+
+filegroup(
+  name = "srcs",
+  srcs = SRCS,
+)
+
+NPM_MODULE_EXTRA_FILES = [
+  "package.json",
+  "README.md",
+]
+
+SRC_DEPS = [
+  "@npm//io-ts",
+  "@npm//lodash",
+  "@npm//tslib",
+  "@npm//fp-ts",
```

Comment:
Can we just order that list as well as `TYPES_DEPS` one? I think it will make it easier to read in the future 😃

Reply:
Yeah, fwiw, I have to constantly save without formatting to keep it at the same indentation level as the others. my IDE (Visual studio) is trying make them formatted at 2 level indentation.

Would be nice if there's a BUILD.bazel linter :-), but yes, for now I will sort these to keep it easier to read. 👍

Reply:
Updated with commit:
https://github.com/elastic/kibana/pull/98999/commits/89b316cbeb758eb2fe0e289bbf11a2e02d3742bf

Reply:
I'm using one for `IntelliJ`. Have you tried that one for vscode? https://marketplace.visualstudio.com/items?itemName=BazelBuild.vscode-bazel

Reply:
Yeah, I'm on that one right now.